src/opt.rs

```diff
@@ -217,6 +217,10 @@ pub struct BuildOpts {
     #[arg(long)]
     pub render_only: bool,
 
+    /// Render the recipe files with solving requirements.
+    #[arg(long, requires("render_only"))]
```

Comment:
nice, didn't know about the `requires`.

Reply:
I was super surprised to see that it's present out-of-the-box in clapy. 
In Click you need to write your own subclass to implement it :(